addon/templates/components/uni-multi-selector.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="input-container">
+  <div class="uni-multi-selector">
+    {{#each options as |option|}}
+      {{uni-selector-button
+        name=name
+        groupValue=groupValue
+        value=option.value
+        label=option.label
```

Comment:
Sorry, but this is an actual label to display, not a key.
I think label and value are more explicit for people who use this component in the future.